resources/definitions/ultimaker.def.json

```diff
@@ -68,6 +68,9 @@
         "skin_material_flow": {
             "value": "0.97 * material_flow"
         },
+        "support_interface_material_flow": {
```

Comment:
Skin material flow will become 95% when the other PP-93 is merged. This is because a top layer is defined to be 100% in that case. Is 95% enough to create a good (closed) interface layer?

Reply:
I think 95% will still give a good enough interface surface. Let's merge it.